metaflow/plugins/kubernetes/kubernetes_decorator.py

```diff
@@ -427,15 +426,15 @@ def task_pre_step(
                 "METAFLOW_KUBERNETES_SERVICE_ACCOUNT_NAME"
             ]
             meta["kubernetes-node-ip"] = os.environ["METAFLOW_KUBERNETES_NODE_IP"]
-            if os.environ.get("METAFLOW_KUBERNETES_JOBSET_NAME"):
-                meta["kubernetes-jobset-name"] = os.environ[
-                    "METAFLOW_KUBERNETES_JOBSET_NAME"
-                ]
+
+            meta["kubernetes-jobset-name"] = os.environ.get(
+                "METAFLOW_KUBERNETES_JOBSET_NAME"
+            )
```

Comment:
todo: add more metadata like: `UBF_CONTEXT` / `RANK` / `WORLD_SIZE`.

Reply:
If we have already set the `CONTROL_TASK_TAG` in metadata then we don't need the `UBF_CONTEXT` ; Check if the `CONTROL_TASK_TAG` is working as expected.